lib/course.rb

```diff
@@ -1 +1,24 @@
+class Course
+  attr_accessor :name, :capacity, :students, :full
+
+  def initialize(name, integer)
+    @name = name
+    @capacity = integer
+    @students = []
+    @full = false
+  end
+
+  def full?
+    if @capacity <= students.count
+      true
+    else
+      false
+    end
+  end
```

Comment:
This works, but could be written more succinctly.
```suggestion
  def full?
    @capacity <= students.count
  end
```